### ISSUE TITLE
[Issue] 0 pokestops found

### ISSUE THREAD

**Issue by linsvatten** [Spegeli/Pokemon-Go-Rocket-API#23]
Hey! it seems to be a problem with my computer because when i run the bot i can´t find any pokestops at all. when i try the same think on another computer it works just fine, now to my question.
Is there any setting on the computer that can prevent the bot from finding pokestops or location in general?

Best regards

**Comment by Spegeli** [comment]
This "bug" still exists? have you tried different accounts on your computer?

**Comment by linsvatten** [comment]
Hey and thanl you for answering. I actually found the problem and it seems like the bot doesn't save my coordinates, it took me a while to notixe this. I fixed it by deleting the debug folder and the put in the coordinates. I have to do this every time i change them. If this is intended or not i don't know.


Btw you've done a awesome job with this bot ;)


Skaffa Outlook för Android






On Sun, Jul 24, 2016 at 7:53 AM +0200, "Spegeli" <notifications@github.com> wrote:





This "bug" still exists? have you tried different accounts on your computer?

---
